src/site/stages/build/plugins/create-drupal-debug.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-param-reassign */
+
+const { ENABLED_ENVIRONMENTS } = require('../../../constants/drupals');
+const { logDrupal: log } = require('../drupal/utilities-drupal');
+
+function createDrupalDebugPage(buildOptions) {
+  if (!ENABLED_ENVIRONMENTS.has(buildOptions.buildtype)) {
+    const noop = () => {};
+    return noop;
+  }
+
+  return (files, smith, done) => {
+    log('Drupal debug page written to /drupal/debug.');
+
+    const drupalPages = Object.keys(files)
+      .filter(fileName => files[fileName].isDrupalPage)
+      .map(fileName => `<li><a href="/${fileName}">${fileName}</a></li>`)
+      .join('');
+
+    const drupalIndex = `
+      <h1>The following pages were provided by Drupal:</h1>
+      <ol>${drupalPages}</ol>
+    `;
+
+    files['drupal/debug/index.html'] = {
+      contents: Buffer.from(drupalIndex),
+      path: '/drupal/debug/',
+      isDrupalPage: true,
+      private: true,
+    };
+
+    done();
+  };
+}
+
+module.exports = createDrupalDebugPage;
```

Comment:
Now that that `/drupal` is dynamic and done in its own plugin, it makes sense for this to be in its own plugin too.